Shared/Extensions/JellyfinAPI/RemoteImageInfo.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2025 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import JellyfinAPI
+import SwiftUI
+
+extension RemoteImageInfo: @retroactive Identifiable {
+
+    public var id: String {
+        UUID().uuidString
```

Comment:
I see this being done often but is an error, since a new `UUID` will be created every time this property is accessed on the same object. Using `hashValue` here is fine.